### PR TITLE
Second release

### PR DESCRIPTION
**Pull request by aamir-unl**

### CODE REVIEW

File: sdk/build.gradle [hunk 1]
```diff
@@ -15,6 +15,9 @@ android {
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
         consumerProguardFiles "consumer-rules.pro"
     }
+//    defaultConfig {
```
Comment (bogdansimon) [inline sdk/build.gradle:18]: can we delete this part?

File: sdk/src/main/java/com/unl/map/sdk/helpers/grid_controls/GridControls.kt [hunk 2]
```diff
@@ -321,4 +331,21 @@ fun <R> CoroutineScope.executeAsyncTask(
         doInBackground() // runs in background thread without blocking the Main Thread
     }
     onPostExecute(result) // runs in Main Thread
+}
+
+fun View.getBitmapFromView(text:String): Bitmap? {
```
Comment (bogdansimon) [inline sdk/src/main/java/com/unl/map/sdk/helpers/grid_controls/GridControls.kt:336]: what is the purpose of this function? Is it used anymore?

File: sdk/src/main/java/com/unl/map/sdk/views/UnlMapView.kt [hunk 3]
```diff
@@ -227,4 +239,35 @@ class UnlMapView @JvmOverloads constructor(
...
+        val symbolOptions = SymbolOptions()
+            .withLatLng(LatLng(30.7120452, 76.8144185))
+            .withIconImage("airport")
+            .withIconSize(1.3f)
+
+        // Use the manager to draw the annotations.
+        symbolManager.create(symbolOptions)
+    }
+
+    private fun addAirplaneImageToStyle(style: Style) {
```
Comment (bogdansimon) [inline sdk/src/main/java/com/unl/map/sdk/views/UnlMapView.kt:264]: what is the purpose of this function?

File: sdk/src/main/java/com/unl/map/sdk/helpers/grid_controls/GridControls.kt [hunk 4]
```diff
@@ -321,4 +331,21 @@ fun <R> CoroutineScope.executeAsyncTask(
...
+    val returnedBitmap =
+        Bitmap.createBitmap(100, 50, Bitmap.Config.ARGB_8888)
+    //Bind a canvas to it
+
+    val canvas = Canvas(returnedBitmap)
+    val paint = Paint()
+    canvas.drawPaint(paint)
+    paint.color = Color.WHITE
+    paint.textSize = 16F
+    canvas.drawText("My Text", android.R.attr.x.toFloat(), android.R.attr.y.toFloat(), paint)
```
Comment (bogdansimon) [inline sdk/src/main/java/com/unl/map/sdk/helpers/grid_controls/GridControls.kt:347]: "My Text"?

File: sdk/src/main/java/com/unl/map/sdk/views/UnlMapView.kt [hunk 5]
```diff
@@ -157,12 +163,17 @@ class UnlMapView @JvmOverloads constructor(
...
                                     ContextCompat.getColor(context,
-                                        R.color.cell_default_color)))
+                                        com.unl.map.R.color.cell_default_color)))
                                 style.addLayer(fillLayer)
                             } catch (e: Exception) {
                                 Log.e(CELL_ERROR, "Error While Adding Grid Cell Source")
                             }
                         }
+                       if(style.getLayer(LayerIDs.CELL_POP_LAYER_ID.name)==null){
+//                           val symbolLayer = SymbolLayer(LayerIDs.CELL_POP_LAYER_ID.name, SourceIDs.CELL_SOURCE_ID.name)
```
Comment (bogdansimon) [inline sdk/src/main/java/com/unl/map/sdk/views/UnlMapView.kt:173]: Do we need this part? Try to cleanup commented out code

File: sdk/src/main/java/com/unl/map/sdk/helpers/grid_controls/GridControls.kt (hunk 2 above)
Comment (aamir-unl) [reply sdk/src/main/java/com/unl/map/sdk/helpers/grid_controls/GridControls.kt:336]: i am updating it today  as it was not working

File: sdk/src/main/java/com/unl/map/sdk/helpers/grid_controls/GridControls.kt (hunk 4 above)
Comment (aamir-unl) [reply sdk/src/main/java/com/unl/map/sdk/helpers/grid_controls/GridControls.kt:347]: it was for testing

File: sdk/src/main/java/com/unl/map/sdk/views/UnlMapView.kt (hunk 5 above)
Comment (aamir-unl) [reply sdk/src/main/java/com/unl/map/sdk/views/UnlMapView.kt:173]: NO i will remove it it was actually for test purpose

**Pull request closed without merging**